astropy/time/core.py

```diff
@@ -2245,13 +2274,37 @@ def to_value(self, unit, equivalencies=[]):
         Returns
         -------
         value : `~numpy.ndarray` or scalar
-            The value in the units specified.
+            The value in the format or units specified.
 
         See also
         --------
         to : Convert to a `~astropy.units.Quantity` instance in a given unit.
         value : The time value in the current format.
         """
+        if unit is None:
+            # Does this look like we're trying to get a TimeDelta format?
+            if format is None or format in self.FORMATS or (
+                    subfmt is not None and not isinstance(subfmt, list)):
+                return super().to_value(format, subfmt)
+
+            # If not, try to interpret as unit and equivalencies.
+            try:
+                unit = u.Unit(format)
+            except ValueError as exc:
+                raise ValueError("format is not one of {} and trying to parse "
+                                 "it led to '{!r}'."
+                                 .format(list(self.FORMATS), exc)) from None
+            if subfmt is not None:
+                # Backwards compatibility.
+                if equivalencies == [] and isinstance(subfmt, list):
+                    equivalencies = subfmt
```

Comment:
Issue a warning about the API change and that the caller should use `equivalencies=[..]`.